src/qute/languageServer/quteServerStarter.ts

```diff
@@ -30,13 +30,19 @@ function prepareParams(): string[] {
     }
   }
 
-  const vmargs = workspace.getConfiguration("quarkus.tools").get("server.vmargs", '');
```

Comment:
Just to confirm, this basically did nothing right ? Unless a user actually defined the property manually, which we didn't really "broadcast" via package.json ..

Reply:
Yes, this did nothing.